client/state/selectors/is-site-on-free-plan.js

```diff
@@ -10,13 +10,13 @@ import { PLAN_FREE } from 'lib/plans/constants';
  *
```

Comment:
Similar to the other selector - the line about `null` return value should be removed.